packages/server/postgres/migrations/1703031300000_addEmbeddingTables.ts

```diff
@@ -0,0 +1,107 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+import {r} from 'rethinkdb-ts'
+import connectRethinkDB from '../../database/connectRethinkDB'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+  DO $$
+  BEGIN
+      IF NOT EXISTS (SELECT 1 FROM pg_type WHERE typname = 'EmbeddingsObjectTypeEnum') THEN
+          EXECUTE 'CREATE TYPE "EmbeddingsObjectTypeEnum" AS ENUM (''retrospectiveDiscussionTopic'')';
```

Comment:
It's an artifact of me wrapping these statements in a transaction. If we don't mind them being outside a transaction I can get rid of the EXECUTE. Given it's a migration, it would be just as safe...

What's your preference?

Reply:
ahhh that explains it! preference is to remove them since other folks will likely look to this file to do something similar in the future, but if time is an issue a comment saying EXECUTE isn't necessary should suffice